tlsutil/config.go

```diff
@@ -30,8 +30,12 @@ var TLSLookup = map[string]uint16{
 	"tls10": tls.VersionTLS10,
 	"tls11": tls.VersionTLS11,
 	"tls12": tls.VersionTLS12,
+	"tls13": tls.VersionTLS13,
 }
 
+// TLSVersions has all the keys from the map above, make sure to keep both in sync
+var TLSVersions = "tls10, tls11, tls12, tls13"
```

Comment:
I didn't want to have a function iterating over the keys again and again. And then I had a var with `[]string`, but given we only ever join, I figured it could be a string as well in the first place.

Reply:
And I think it is better now because both vars are colocated and it is easier to spot the dependency than before.

Reply:
```
var TLSVersions string
func init() {
   var versions []string
   for version, _ := range TLSLookup {
      versions = append(versions, version)
   }
   TLSVersions = strings.Join(versions, ", ")
}
```

Reply:
That should do a one time generation of the string. But like I said, it doesn't really matter. I doubt there will be a tls 1.4 any time soon. Its sort of like the copyright thing. You might change it once a year so is it worth keeping the code around rather than just updating it very rarely?